H4API/lib/h4api/H4Base/H4Base.cpp

```diff
@@ -77,7 +77,7 @@ namespace h4
 		case 0x31D600:
 			m_version = GameVersion::LEGACY_PATCHED_EDITOR;
 			break;
-		case 0x56FEC0:
+		case 0x56FECB:
```

Comment:
??
![image](https://user-images.githubusercontent.com/25070750/211160345-69b2cb06-58cc-4441-913e-665c7a37264e.png)

Reply:
I used your patcher and got "unsupported exe" error until this change.

Reply:
Ahhh, right - was looking at a more recent patcher that I never published. Using the one in the repo you are correct.